### PR TITLE
fix: first code group item is not open by default

### PR DESCRIPTION
**Pull request by dcdunkan**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Perhaps the site will be fixed, we should wait

**Comment by dcdunkan** [comment]
> Perhaps the site will be fixed, we should wait

Any updates?

**Comment by KnorpelSenf** [comment]
None that I am aware of

File: iv-rules.txt [hunk 1]
```diff
@@ -46,9 +46,9 @@ body: //main
 @map( //details[has-class("code-group-item")] ) {
   $details: $@
   @prepend_to($details): (//ul[@class="code-group__ul"]/summary)[position()<2]
-  @if ($details[has-class("code-group-item__active")]) {
+  @if ( $details[@aria-selected="true"] ) {
```
Comment (CikiMomogi) [inline iv-rules.txt:49]:
```suggestion
  @if ( $details//button[@ariaexpanded="true"] ) {
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Comment by KnorpelSenf** [comment]
@roj1512 could you chip and let us know if this should be merged or if the website will be fixed instead?

**Review by CikiMomogi — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]

**Pull request merged**